From 77a52f8866c8a07c776ee1f1ed29717e3f02de65 Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Tue, 6 Sep 2022 07:43:11 +0800
Subject: [PATCH 04/60] Revert "drm/vc4: change vc4_dma_range_matches from a
 global to static"

This reverts commit a60996dc027a026baca91a5bbf948bd3fb30ecd1.
---
 drivers/gpu/drm/vc4/vc4_drv.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/vc4/vc4_drv.c b/drivers/gpu/drm/vc4/vc4_drv.c
index 52426bc8edb8b..4bed62916b648 100644
--- a/drivers/gpu/drm/vc4/vc4_drv.c
+++ b/drivers/gpu/drm/vc4/vc4_drv.c
@@ -246,7 +246,7 @@ static void vc4_match_add_drivers(struct device *dev,
 	}
 }
 
-static const struct of_device_id vc4_dma_range_matches[] = {
+const struct of_device_id vc4_dma_range_matches[] = {
 	{ .compatible = "brcm,bcm2711-hvs" },
 	{ .compatible = "brcm,bcm2835-hvs" },
 	{ .compatible = "brcm,bcm2835-v3d" },
-- 
2.25.1

